lambda/health_package/metric-settings.json

```diff
@@ -1,4 +1,206 @@
 [
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "IncomingBytes",
+    "Statistic": "Average",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 599,
+    "Description": "The number of bytes successfully put to the Kinesis stream over the specified time period. This metric includes bytes from PutRecord and PutRecords operations. Minimum, Maximum, and Average statistics represent the bytes in a single put operation for the stream in the specified time period."
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "IncomingRecords",
```

Comment:
I'm not sure about this absolute volume metrics. Our traffic is very variable outside hours / weekends / holidays. I think we're better off sticking to the measures that detect whether the stream is keeping up with whatever the demand is like latency.

Reply:
Fair enough, will remove.